core/arch/x86/x86.asm

```diff
@@ -2242,45 +2242,45 @@ GLOBAL_LABEL(get_ymm_caller_saved:)
         * portable or cause frequency scaling (i#3169). We just put in the raw bytes
         * for these instrs:
         * Note the 64/32 bit have the same encoding for either rax or eax.
-        * c5 fe 7f 00               vmovdqu %ymm0,0x00(%xax)
-        * c5 fe 7f 48 20            vmovdqu %ymm1,0x20(%xax)
-        * c5 fe 7f 50 40            vmovdqu %ymm2,0x40(%xax)
-        * c5 fe 7f 58 60            vmovdqu %ymm3,0x60(%xax)
-        * c5 fe 7f a0 80 00 00 00   vmovdqu %ymm4,0x80(%xax)
-        * c5 fe 7f a8 a0 00 00 00   vmovdqu %ymm5,0xa0(%xax)
+        * c5 fe 7f 00               vmovdqu %ymm0,0x00(%rax)
```

Comment:
Shouldn't this remain `xax` b/c it's `eax` in 32-bit?  DItto below.

Reply:
Yes. Fixed. Also in get_caller_zmm_saved.